src/pages/dar_application/ResearcherInfo.js

```diff
@@ -228,7 +228,7 @@ export default function ResearcherInfo(props) {
               }),
               span({
                 className: 'cancel-color required-field-error-span', isRendered: (invalidInvestigator) && (showValidationMessages)
-              }, ['Required field'])
+              }, ['Required field, ensure your Profile has a PI listed or marks you as the PI'])
```

Comment:
Minor language nit, the researcher would have to mark themselves as a PI. This reads as if a PI must do something for the researcher. 

```suggestion
              }, ['Required field, ensure your Profile has a PI listed, or that you are selected as the PI'])
```